lib/active_merchant/billing/gateways/trans_first_transaction_express.rb

```diff
@@ -377,20 +473,24 @@ def parse(xml)
       end
 
       def success_from(response)
+        return unless response
+
         fault = response['Fault']
         approved_transaction = APPROVAL_CODES.include?(response['rspCode'])
-        found_contact = response['FndRecurrProfResponse']
+        found_contact = response['FndRecurrProfResponse'] || response['cust']
```

Comment:
This is a bit different then: https://github.com/rednovalabs/active_merchant/pull/9/commits/9fb6335bac7f5331f831f88fc19b518b43b1049b